src/System.Private.CoreLib/shared/System/Threading/Thread.cs

```diff
@@ -179,7 +119,7 @@ public CultureInfo CurrentUICulture
             {
                 if (this != CurrentThread)
                 {
-                    SetCultureOnUnstartedThread(value, ref _startUICulture);
+                    SetCultureOnUnstartedThread(value, true);
```

Comment:
Can you use named arguments for these Booleans so that it's more obvious at the call site what they mean?

Reply:
Will do.